src/components/shared/VideoUploader.tsx

```diff
@@ -117,14 +117,14 @@ export class VideoUploader extends React.Component<
     // }
 
     // const metadata = {
-    //   //@ts-ignore Expo Blob does not have data type
+    //   // (@)ts-ignore Expo Blob does not have data type
     //   contentType: blob.data.type
     // };
 
-    const videoUploadTask = videoUploadRef.put(videoUri, {
+    const videoUploadTask = videoUploadRef.putFile(videoUri, {
```

Comment:
`putFile` is the proper method to call on `react-native-firebase` references. tested by uploading a verification video.

Reply:
lol